taskweaver/module/tracing.py

```diff
@@ -0,0 +1,156 @@
+from typing import Literal, Optional
+
+from injector import inject
+
+from taskweaver.config.module_config import ModuleConfig
+
+
+class TracingConfig(ModuleConfig):
+    def _configure(self):
+        self._set_name("tracing")
+        self.enabled = self._get_bool("enabled", False)
+        self.endpoint = self._get_str("endpoint", "http://127.0.0.1:4318/v1/traces")
+        self.service_name = self._get_str("service_name", "taskweaver.opentelemetry.tracer")
+        self.exporter = self._get_str("exporter", "otlp")
```

Comment:
what else value can this be configured?

Reply:
I see the "console", so the trace would be printed directly to console? then Jaeger is not needed to display the traces? sounds like this is more lightweight than the "otlp" option

Reply:
Console is purely for debugging and will mess up the output. So it is not a real option. This configure is not supposed to be exposed to users.